src/ess/reflectometry/orso.py

```diff
@@ -32,102 +32,102 @@
     TaskGraph = Any
 
 
-OrsoCreator = NewType('OrsoCreator', orso_base.Person)
+OrsoCreator = NewType("OrsoCreator", orso_base.Person)
 """ORSO creator, that is, the person who processed the data."""
 
-OrsoDataSource = NewType('OrsoDataSource', data_source.DataSource)
+OrsoDataSource = NewType("OrsoDataSource", data_source.DataSource)
 """ORSO data source."""
 
-OrsoExperiment = NewType('OrsoExperiment', data_source.Experiment)
+OrsoExperiment = NewType("OrsoExperiment", data_source.Experiment)
 """ORSO experiment for the sample run."""
 
-OrsoInstrument = NewType('OrsoInstrument', data_source.InstrumentSettings)
+OrsoInstrument = NewType("OrsoInstrument", data_source.InstrumentSettings)
 """ORSO instrument settings for the sample run."""
 
-OrsoIofQDataset = NewType('OrsoIofQDataset', orso.OrsoDataset)
+OrsoIofQDataset = NewType("OrsoIofQDataset", orso.OrsoDataset)
 """ORSO dataset for reduced I-of-Q data."""
 
-OrsoMeasurement = NewType('OrsoMeasurement', data_source.Measurement)
+OrsoMeasurement = NewType("OrsoMeasurement", data_source.Measurement)
 """ORSO measurement."""
 
-OrsoOwner = NewType('OrsoOwner', orso_base.Person)
+OrsoOwner = NewType("OrsoOwner", orso_base.Person)
 """ORSO owner of a measurement."""
 
-OrsoReduction = NewType('OrsoReduction', reduction.Reduction)
+OrsoReduction = NewType("OrsoReduction", reduction.Reduction)
 """ORSO data reduction metadata."""
 
-OrsoSample = NewType('OrsoSample', data_source.Sample)
+OrsoSample = NewType("OrsoSample", data_source.Sample)
 """ORSO sample."""
 
 
-def parse_orso_experiment(filepath: FilePath[Sample]) -> OrsoExperiment:
+def parse_orso_experiment(Filename: Filename[SampleRun]) -> OrsoExperiment:
```

Comment:
(minor) should be lowercase right?

Reply:
I think it's a bad find/replace in the editor...